projects/npm-tools/packages/npm-scripts/src/scripts/build.js

```diff
@@ -70,6 +71,19 @@ function runBridge() {
 	});
 }
 
+// Utility for getting paths to @clayui/css variables
+// This shouldn't ever fail, but is necessary so that we don't require
+// '@clayui/css' as a dependency in this package.
+
+const getClayPaths = () => {
+	try {
+		return require('@clayui/css').includePaths;
+	}
+	catch (e) {
```

Comment:
I was going to say [don't abbreviate](https://github.com/liferay/liferay-frontend-guidelines/blob/master/general/naming.md#avoid-abbreviations), but you can just delete the whole `(e)`, I think, because AFAIK, [optional catch binding](https://v8.dev/features/optional-catch-binding) works in Node 10, which is the minimum version we're using, realistically.